database-mysql/schema.sql

```diff
@@ -0,0 +1,14 @@
+DROP DATABASE IF EXISTS calendar;
+
+CREATE DATABASE calendar;
+
+USE calendar;
+
+-- Table will hold dates that a specific place is unavailable
+-- and the minimum number of days someone has to stay to reserve the location
+CREATE TABLE availabilityDays(
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    startDate DATE,
+    endDate DATE,
+    minDaysStay INT
+) ;
```

Comment:
Are you generating segments of dates that are available to book? Or segments of dates that have been booked? Also, how do you map these to a space now that you're going to a single table?

Reply:
generating segments of dates that are already booked. Also I am just doing one segment of dates per property so no need to be mapped. Since my unique id will run from 1 - 100, each of those will represent one location